File: src/dev/4.5/Legacy.EFLViewModelBase/Attributes/MvvmViewModelAttribute.vb
''' <summary>
''' Bestimmt für ein ViewModel die standardmäßig für Windows Forms zu verwendende View.
''' </summary>
''' <remarks></remarks>
Public NotInheritable Class MvvmViewModelAttribute
    Inherits Attribute

    ''' <summary>
    ''' Erstellt eine neue Instanz dieser Attribut-Klasse, die eine Klasse als ViewModel kennzeichnet.
    ''' </summary>
    ''' <remarks></remarks>
    Sub New()
        MyBase.New()
    End Sub

End Class

File: src/dev/MvvmBaseLib/MvvmBaseLibShared/Attributes/MvvmViewModelAttribute.cs
using System;
/// <summary>
/// Bestimmt für ein ViewModel die standardmäßig für Windows Forms zu verwendende View.
/// </summary>
/// <remarks></remarks>
public sealed class MvvmViewModelAttribute : Attribute
{
    /// <summary>
    /// Erstellt eine neue Instanz dieser Attribut-Klasse, die eine Klasse als ViewModel kennzeichnet.
    /// </summary>
    /// <remarks></remarks>
    public MvvmViewModelAttribute() : base()
    {
    }

}
